.github/workflows/nightly_kernel_check.yml

```diff
@@ -0,0 +1,28 @@
+name: Nightly Kernels Check
+
+on:
+  pull_request:
+  schedule:
+    - cron: "0 2 * * *"
+  workflow_dispatch:
+
+jobs:
+  kernels-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install git+https://github.com/huggingface/kernels kernel-abi-check
```

Comment:
`kernels check` requires the `kernels` installation to be from `main`. Okay for now I guess?

Reply:
Yeah, I don't think that's a problem.